docs/docs/additional_resources/dependents.mdx

```diff
@@ -19,7 +19,7 @@ Dependents stats for `langchain-ai/langchain`
 |[moymix/TaskMatrix](https://github.com/moymix/TaskMatrix) | 34342 |
 |[geekan/MetaGPT](https://github.com/geekan/MetaGPT) | 31126 |
 |[streamlit/streamlit](https://github.com/streamlit/streamlit) | 28911 |
-|[reworkd/AgentGPT](https://github.com/reworkd/AgentGPT) | 27833 |
+|[reworked/AgentGPT](https://github.com/reworkd/AgentGPT) | 27833 |
```

Comment:
this is the name of the repo, shouldn't be changed

Reply:
Ok

Reply:
Resolved